x/concentrated-liquidity/incentives.go

```diff
@@ -79,6 +79,7 @@ func (k Keeper) GetUptimeAccumulatorValues(ctx sdk.Context, poolId uint64) ([]sd
 // This value depends on the provided tick's location relative to the current tick. If the provided tick
 // is greater than the current tick, then the value is zero. Otherwise, the value is the value of the
 // current global spread reward growth.
+// TODO: EXPLAIN THAT CORRECTNESS HERE REQUIRES A NUANCED ARGUMENT
```

Comment:
What do you mean by this? We need to resolve all questions below?

Reply:
Nah I'll come back and add these, those were mostly notes to self.

The nuanced argument here is that this is true iff this is being called for consecutive ticks, not if we were jump around.